src/main/java/practice/StreamPractice.java

```diff
@@ -59,7 +89,12 @@ public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
      * return the names of all cats whose owners are women from `femaleAge` years old inclusively.
      */
     public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> p.getAge() > femaleAge && p.getSex() == Person.Sex.WOMAN)
+                .map(Person::getCats)
+                .flatMap(Collection::stream)
```

Comment:
```suggestion
                .flatMap(person -> person.getCats().stream())
```
use just 1 map instead of 2 when possible